cosmoz-templatizer.html

```diff
@@ -33,28 +33,13 @@
 			}
 
 			this.dataHost = owner;
-			this._patchConstructorImpl();
+
 			Polymer.Templatizer.templatize.call(this, template);
 			template.__templatizer = this;
 
-			return this.ctor;
-		},
-
-		_patchConstructorImpl() {
-			let _ctor  = this._constructorImpl,
-				self = this;
-
-			this._constructorImpl = function (model = {}, host = self) {
-				if (self._parentProps) {
-					var templatized = self._templatized;
-					for (let prop in self._parentProps) {
-						if (model[prop] === undefined) {
-							model[prop] = templatized[self._parentPropPrefix + prop];
-						}
-					}
-				}
-				return _ctor.call(this, model, host);
-			};
+			return (function (self, model){
```

Comment:
Should we return the same thing at the beginning of the `templatize` function, if `ctor` is defined ?

Reply:
Ah yes. I'll update that.